### DISCUSSION TITLE
Resolution as data exploration

### DISCUSSION THREAD

**Discussion by RichardScottOZ** [gjoseph92/stackstac#19] [General]
After getting another download failure type error

I realised this had enabled with a very simple to use resolution keyword option (as opposed to doing that all yourself each time) the ability to ask for a much lower resolution of the same thing quickly, as a check to see if it was doing what you want.

e.g. 
![image](https://user-images.githubusercontent.com/72196131/112737651-f0f99480-8fab-11eb-98ef-53db994c84b9.png)

Does it cover everywhere you want?  No.

**Comment by gjoseph92** [comment]
Yes, starting with a very coarse resolution just for data exploration can be a great way to sanity-check things (assuming the data is in COGs or some other format with overviews!). Did you find the runtimes acceptable at low resolution over a large area?

**Comment by RichardScottOZ** [comment]
On a laptop doing a 1.5mkmsq area at a 4 figure resolution (at dodgy hfc NBN over wifi), for 200 odd timesteps, was a 'go and have lunch' kind of time - so pretty good, really, yes.  Probably minutes on AWS - will try it sometime later in the week.  Trialling a 1000+ timestep over the same area now. I imagine it should increase scale a little - as not just more timesteps but more coverage as increase the cloud coverage percentage.
